modules/rpc/handler.go

```diff
@@ -34,6 +35,8 @@ type contextInboundMiddleware struct {
 }
 
 func (f contextInboundMiddleware) Handle(ctx context.Context, req *transport.Request, resw transport.ResponseWriter, handler transport.UnaryHandler) error {
+	stopwatch := stats.RPCHandleTimer.Timer("rpc.handler." + req.Procedure + ".time").Start()
```

Comment:
I am following muttleys naming schemes. Having "rpc.handler.procedure" should be fine for query. We are not using prometheus right now. We should fix how we define scope however, in other news...